src/PowerShell/Microsoft.WinGet.Client.Cmdlets/Cmdlets/InstallerSelectionCmdlet.cs

```diff
@@ -0,0 +1,55 @@
+﻿// -----------------------------------------------------------------------------
+// <copyright file="InstallerSelectionCmdlet.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------------
+
+namespace Microsoft.WinGet.Client.Commands
+{
+    using System.Management.Automation;
+    using Microsoft.WinGet.Client.Commands.Common;
+    using Microsoft.WinGet.Client.PSObjects;
+
+    /// <summary>
+    /// This is the base class for all commands that select and installer from a given package.
```

Comment:
select an installer? or select and installs?

Reply:
Fixed. Should be `select an installer`